kornia/augmentation/container/augment.py

```diff
@@ -169,6 +167,14 @@ def __init__(
             random_apply_weights=random_apply_weights,
         )
 
+        self._parse_transformation_matrix_mode(transformation_matrix_mode)
+
+        self._valid_ops_for_transform_computation = (
+            K.RigidAffineAugmentationBase2D,
+            K.RigidAffineAugmentationBase3D,
```

Comment:
i think is better fully import to avoid imports issues

Reply:
+1